### DISCUSSION TITLE
fix padding on small design

### DISCUSSION THREAD

**Discussion by MohamedAli00949** [tailwindlabs/tailwindcss#12728] [Help] [answered]
**What version of Tailwind CSS are you using?**
v3.3.0

**What build tool (or framework if it abstracts the build tool) are you using?**
postcss-cli v8, Next.js 14.04

**What version of Node.js are you using?**
v20.10.0

**What browser are you using?**
Chrome, Brave

**What operating system are you using?**
Windows

**Reproduction URL**
[Component URL](https://github.com/entagk/entagk.com/blob/master/app/footer.tsx)

**Describe your issue**
I want to add padding with `0.5rem` value in min-width `320px` so I use `min-[320px]:px-2` but this is didn't happen
and I have tried to add it `min-[320px]:px-[0.5rem]` but getted the same result.

**Comment by adamwathan** [comment, marked as answer]
Hey! This works as I'd expect here:

https://play.tailwindcss.com/vmpo5HNEhF?size=360x720

No padding when the viewport is < 320px, and 0.5rem of padding when the viewport is at least 320px wide.

**Reply by MohamedAli00949** [reply]
I am sorry, this didn't work in the development environment because of some of the issues with my machine.
Thank you
